src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -160,13 +164,24 @@ def format_historical_data(
     return format_snuba_ts_data(data, query_columns, organization)
 
 
+def get_dataset_from_label(dataset_label: str):
+    if dataset_label == "events":
+        # DATASET_OPTIONS expects the name 'errors'
+        dataset_label = "errors"
+    elif dataset_label in ["generic_metrics", "transactions"]:
+        # XXX: performance alerts dataset differs locally vs in prod
+        dataset_label = "discover"
+    return get_dataset(dataset_label)
```

Comment:
is the default case that we want to pass through to here? will this throw if it's an unexpected dataset?

Reply:
If it's unexpected something is seriously wrong with how we're creating alert rules, but I can throw something to be safe